src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -768,7 +770,11 @@ private static BundleEntryComponent encounter(Person person, BundleEntryComponen
           .setSystem("https://github.com/synthetichealth/synthea")
           .setValue(encounterResource.getId());
     }
-    return entry;
+
+    if (encounter.additionalAttributes != null) {
+      entry.setResource(setAdditionalAttributes(encounterResource, encounter.additionalAttributes));
+    }
+      return entry;
```

Comment:
NIT: This should be un-indented 2 spaces
```suggestion
    return entry;
```